ANONYMITY_NETWORKS.md

```diff
@@ -165,15 +162,15 @@ simply a best effort attempt.
 ### Active Bandwidth Shaping
 
 An attacker could attempt to bandwidth shape traffic in an attempt to determine
-the source of a Tor/Kovri/I2P connection. There isn't great mitigation against
-this, but Kovri/I2P should provide better protection against this attack since
+the source of a Tor/I2P connection. There isn't great mitigation against
+this, but I2P should provide better protection against this attack since
 the connections are not circuit based.
 
 #### Mitigation
 
-The best mitigiation is to use Kovri/I2P instead of Tor. However, Kovri/I2P
+The best mitigiation is to use I2P instead of Tor. However, I2P
 has a smaller set of users (less cover traffic) and academic reviews, so there
 is a tradeoff in potential isses. Also, anyone attempting this strategy really
 wants to uncover a user, it seems unlikely that this would be performed against
-every Tor/Kovri/I2P user.
+every Tor/I2P user.
```

Comment:
You have a lot of diffs that just remove "Kovri" for no apparent reason.

Reply:
Kovri is an implementation, this is supposed to work with any i2p implementation. Just like Tor is not calling out particular implementations.

Reply:
Good. I feared it might have been spite :)